app/views/layouts/application.html.erb

```diff
@@ -11,35 +11,35 @@
                      type: "image/png" %>
     <%= content_for :social_media_meta_tags %>
 
-    <%= setting["html.per_page_code_head"].try(:html_safe) %>
+    <%= raw setting["html.per_page_code_head"] %>
   </head>
   <body class="<%= yield (:body_class) %>">
-    <%= setting["html.per_page_code_body"].try(:html_safe) %>
+    <%= raw setting["html.per_page_code_body"] %>
```

Comment:
We're leaving it like this for now, since this setting can only be changed by administrators and removing the code wouldn't allow administrators to add custom scripts.

Reply:
We might change it in the future, though :thinking:.